tests/test_operator.py

```diff
@@ -30,3 +30,42 @@ def test_my_operator():
     )
     ti = TaskInstance(task=task, execution_date=datetime.now())
     task.execute(ti.get_template_context())
+
```

Comment:
Add blank line before `test_operator_fail` and `test_operator_fail_invalid_tier` and at end of file